public/create.html

```diff
@@ -5,7 +5,8 @@
   <head>
   	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
   	<title>create a new pile</title>
-    <link rel="stylesheet" href="/style.css">
+    <link href="/auto.css?d=20210524" type="text/css" rel="stylesheet">
+    <link href="/custom.css" type="text/css" rel="stylesheet">
```

Comment:
one last thing. i think you missed  the cachebreaker on the custom.css

Reply:
added